test/unitTest/java/org/thoughtcrime/securesms/translation/TranslationTaskUnitTest.java

```diff
@@ -0,0 +1,73 @@
+package org.thoughtcrime.securesms.translation;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.thoughtcrime.securesms.BaseUnitTest;
+import org.thoughtcrime.securesms.database.model.MessageRecord;
+
+import java.io.IOException;
+
+import okhttp3.Call;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+import static junit.framework.Assert.assertEquals;
+import static junit.framework.Assert.fail;
+import static org.powermock.api.mockito.PowerMockito.mock;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({ OkHttpClient.class, Request.class, Response.class, Call.class })
+public class TranslationTaskUnitTest extends BaseUnitTest {
```

Comment:
Could possibly add in a unit test to the conversation fragment to make sure that the translation task is called upon handling a translation